src/webots/nodes/WbRobot.cpp

```diff
@@ -158,7 +158,18 @@ WbRobot::WbRobot(WbTokenizer *tokenizer) : WbSolid("Robot", tokenizer) {
   init();
 }
 
-WbRobot::WbRobot(const WbRobot &other) : WbSolid(other) {
+WbRobot::WbRobot(const WbRobot &other) :
+  WbSolid(other),
+  mControllerDir(),
```

Comment:
All these initialization lines are useless and should be removed because QString are already initialized by default.

Reply:
cppcheck triggers an error if they are missing.

Reply:
Ok. But the cppcheck error/warning is wrong.

Reply:
I don't know if using a lot of suppress tags for cppcheck is really the best solution. Is it really that bad to initialize those QString objects?

Reply:
It is completely useless to explicitly initialize the strings and all the other Qt objects
From my point of view it makes the code less readable than before because we have a long list of useless instructions.
I think that cppcheck should help us writing better code, not worse and lately it is not always the case.
We should try to adjust the configuration to try to avoid these false positives.